compiler/damlc/tests/daml-test-files/ChoiceShadowing.daml

```diff
@@ -24,13 +25,10 @@ template T with
           p = p
           p2 = p2
 
-test1 = scenario do
-    alice <- getParty "Alice"
-    bob <- getParty "Bob"
-    cid <- submit alice $ create (T alice bob)
-    submit bob do
-      cid2 <- exercise cid (Call bob)
-      newT <- fetch cid2
-      newT === T bob bob
-
--- @ENABLE-SCENARIOS
```

Comment:
Notice the choice argument has the same name as one of the template arguments, we're testing that A: It doesn't error with shadowing and B: it uses the choice value, rather than the template value, within a choice body